terraform/aks/application.tf

```diff
@@ -19,6 +19,7 @@ module "web_application" {
   probe_path    = var.probe_path
 
   web_external_hostnames = var.gov_uk_host_names
+  enable_logit           = var.enable_logit
```

Comment:
should be set in any worker deployments too

Reply:
Added enable_logit to worker_application